### DISCUSSION TITLE
Approval of business plan change -- Deemed no objection

### DISCUSSION THREAD

**Discussion by kennethng-ledn** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#38] [General]
The proposed requirement for VASPs to obtain prior written approval of the Authority to make a change to the approved 
business plan would introduce friction for VASPs looking to innovate quickly.  Given the pace of development in the virtual asset space, VASPs require the ability to stay nimble and pivot quickly in order to stay competitive in the market place.

One suggestion is to introduce a timeline for deemed no objection to a business plan change after a reasonable period after the submission of the change (15 days to 30 days).  This would provide VASPs with the certainty that it needs to move forward with changes, while affording the Authority a reasonable period of time to review proposed changes and put forth any comments.

**Comment by Aaron-XReg** [comment]
I agree with this suggestion @kennethng-ledn. The deemed no objection approach is common internationally and allows regulators to conserve resources rather than having to actively approve all changes of business plans. Where the regulator does have an objection or require further information 30 days would be plenty of time to identify any issues.

**Comment by AGIAI** [comment]
@kennethng-ledn This approach makes much more sense. 15 days seems a sensible time in which to file an objection

**Comment by FinalTwist** [comment]
I looked at this requirement... the Insurance Act has the same requirement for an approval prior to any change to the business plan.  There is precedent here so I assume that's why that requirement was added.  Why would it work for the insurance industry, but not for the VASP sector?

**Reply by kennethng-ledn** [reply]
@FinalTwist I would argue that the core insurance business model (other than certain innovations at the periphery) is centuries-old, while insurance industry laws and regulations are decades-old. Insurance industry business models are much more mature as a result.
 
Contrast that against the entire VASP sector which is still in its nascent stage, let alone the _regulated_  VASP sector which is just coming into place and still reacting to new laws and regulations which are being formed in real time (I can think of no better example than this live consultation process on Github which I think is groundbreaking!) Consider that together with the underlying pace of innovation in this space (virtual, borderless, 24/7), we are seeing business models in the VASP sector, at least at this moment in time, evolving at tremendous speed.  

For these reasons, I do believe that a deemed no objection approach as outlined in the thread above is crucial for the development of this sector, at least at this moment in time (which approach could be reconsidered as the sector matures).

**Reply by Zhira7** [reply]
Wouldn’t the nascent nature of the crypto industry actually necessitate the need for significant business change approval ? While proposing a date limit is not a bad idea, there’s also need to consider how realistic that is for example  if a VASP notifies on Dec 24th, a 15 day turnaround is not practical. If the insurance industry has a broader runway, crypto should too and possibly a longer one.

**Reply by kennethng-ledn** [reply]
Hi @Zhira7 Where the regulator does have an objection or require further information, they can raise the issue within such period and the participant would not proceed. The deemed no objection would only apply in the absence of any feedback from the regulator.

**Comment by Cayman-Blockahain-Association** [comment]
Based on consultation with our members we agree with all comments raised above. Given the pace of change in the industry prior written approval would not be feasible. The deemed no objection approach within 15 days was largely supported. 

Additionally some members suggested that limited pre-notification items could specifically be identified if deemed necessary.

**Comment by CaraProvenance** [comment]
Would also add that some business plan changes, particularly in new businesses with evolving regulatory requirements, may not have the luxury of waiting for an approval, but rather be required to ensure business continuity, risk mitigation efforts etc.

**Comment by lkoseland** [comment]
The virtual asset space moves quickly and to keep ahead of competition, the business plan of the entities may need updating from time to time.  To ensure that VASPs are allowed to innovate in the market, I agree with the proposal that no objection within 15 days from the date of submission to the Authority would provide the Authority with adequate time to respond if they had an issue with the revised business plan. 

Alternatively, if the amendments to the revised business plan are not material, it is suggested that there is a notification obligation within [21] days to the Authority is rather than prior approval of the revised business plan. The proposed notification period is in line with what the Authority requires for investment funds registered with the Authority. The materiality could be assessed in the same way it is assessed for updating an offering memorandum for mutual funds. The categories contained in an MF1 form could be used as a guide for what constitutes material changes to a business plan i.e. change of directors, service providers, strategy...
